spring-mvc/src/main/resources/logback.xml

```diff
@@ -29,7 +29,7 @@
     <logger name="org.hibernate.tool.hbm2ddl.SchemaExport" level="off"/>
     <logger name="org.hibernate.engine.jdbc.spi.SqlExceptionHelper" level="off" />
     <logger name="org.hibernate" level="error"/>
-    <logger name="org.springframework.web" level="error"/>
+    <logger name="org.springframework.web" level="debug"/>
```

Comment:
I thought it would be useful for others, as we are all focusing on the web part now, that's why I pushed it. Don't you agree?